src/task/ReceiveVideoInputTask.ts

```diff
@@ -56,6 +56,9 @@ export default class ReceiveVideoInputTask extends BaseTask {
     this.context.activeVideoInput = videoInput;
     if (videoInput) {
       const videoTracks = videoInput.getVideoTracks();
+      if (!videoTracks || videoTracks.length === 0) {
```

Comment:
Can you mention the race condition here in a comment, since its kinda an unusual check (and most people don't like to `git blame` -> read PRs :P.  Also if that race condition occurs we will eventually recover right?  even if we no-opped here.